src/Cassandra.IntegrationTests/Linq/LinqRealClusterTests.cs

```diff
@@ -368,6 +368,31 @@ public void Linq_Writetime_Returns_ValidResult()
             Assert.Less(recordsArr[0].wt2, nowPlus5Minutes);
         }
 
+        [Test]
+        public void Linq_Writetime_Returns_Null()
+        {
+            var writetimeEntityTableName = "writetime_entities";
+            Session.Execute($"CREATE TABLE IF NOT EXISTS {writetimeEntityTableName} (id uuid primary key, propertyint int, propertystring text)");
+
+            var table = new Table<WritetimeEntity>(Session, new MappingConfiguration().Define(
+                new Map<WritetimeEntity>().TableName(writetimeEntityTableName)));
+            var id = Guid.NewGuid();
+            var writetimeEntity = new WritetimeEntity()
+            {
+                Id = id,
```

Comment:
Change the type of `WritetimeEntity.PropertyInt` to `int?` otherwise the driver will send a `0` and `writetime()` will return a valid timestamp (which is why this test is failing atm).

Reply:
Changed, thanks 👍